jwql/website/apps/jwql/views.py

```diff
@@ -226,8 +226,17 @@ def archived_proposals_ajax(request, inst):
     min_obsnums = []
     total_files = []
     proposal_viewed = []
+    proposal_exp_types = []
+    thumb_exp_types = []
+
+    # Get a set of all exposure types used in the observations associated with this proposal
+    exp_types = [exposure_type for observation in all_entries for exposure_type in observation.exptypes.split(',')]
+    exp_types = sorted(list(set(exp_types)))
+
     # The naming conventions for dropdown_menus are tightly coupled with the code, this should be changed down the line.
```

Comment:
Could you expand on this comment in a new github issue?

Reply:
yup!